odoo/tests/common.py

```diff
@@ -470,3 +470,40 @@ def can_import(module):
         return False
     else:
         return True
+
+
+def set_tags(*tags):
+    """
+    A decorator to tag TestCase objects
+    Tags are stored in a set that can be accessed from a 'test_tags' attribute
+    Tags must not contains '+' or '-' signs
```

Comment:
Maybe validate that?

Reply:
So tags can contains spaces, punctuation characters, etc.  They can even be empty.
Isn't it a bit too liberal?  I would restrict them to alphanumeric characters only, for simplicity.

Reply:
Naming suggestion: decorator names are usually not verbs, but rather names or adjectives, more like attribute names.
Why not using `@tagged(...)`?